ui/src/components/Nav/Mobile/index.tsx

```diff
@@ -74,34 +88,49 @@ const Mobile: React.FC<Props> = (props): React.ReactElement => {
-                                                                                </span>
-                                                                            </Components.Link>
-                                                                        ) : (
-                                                                            <Components.Link
-                                                                                href="#"
-                                                                                onClick={handleClose}
-                                                                            >
-                                                                                {subItem}
-                                                                            </Components.Link>
-                                                                        )}
-                                                                    </div>
+                                                                    {({ active }) => (
+                                                                        <Components.Link
+                                                                            href={subItem.value}
+                                                                            onClick={(e) => {
+                                                                                if (
+                                                                                    typeof subItem.onClick ===
+                                                                                    'function'
+                                                                                ) {
+                                                                                    subItem.onClick(e);
+                                                                                }
+
+                                                                                handleClose();
+                                                                            }}
+                                                                            className={`${
+                                                                                active
+                                                                                    ? 'ring-yellow-400 hover:ring-transparent active:ring-yellow-400 '
+                                                                                    : 'ring-transparent'
+                                                                            } text-white m-0 block w-full rounded-md p-1 ring-2`}
+                                                                        >
+                                                                            {subItem.label}
+                                                                        </Components.Link>
```

Comment:
Sorry to be a pain but I've been thinking about how the user navigates around the mobile nav bar using their keyboard - when they use the dropdown menu they navigate down by tabbing but when navigating the option with sub items they have to use the arrow keys. It feels a bit incoherent and may be confusing for the user?

@florin-holhos you made a valid point in standup about redesigning the mobile nav bar so I don't know if it's worth addressing in this PR if we will be re-doing it anyways (in which case I think we will need to inform product to make create a ticket?)